cmd/sql/flow.go

```diff
@@ -158,6 +160,12 @@ func extendLocalCmdArgsWithFlags(cmd *cobra.Command, args []string) []string {
 		if dataDir != "" {
 			args = append(args, "--data-dir", dataDir)
 		}
+		if verbose {
+			args = append(args, "--verbose")
+		}
```

Comment:
for me no-verbose is just else condition for verbose?
cc @neel-astro wdyt?

Also may be we can rename it to quite? As other cli tools.

Reply:
Thank you @andriisoldatenko for reviewing. The flags need to be the same as in the sql-cli. So that we don't have to document the flags twice. The help will be provided by the sql-cli.

Reply:
ah okey.

Reply:
We are trying to avoid any overhead on the cobra/astro-cli side. For example, we are also not validating the cli commands. This also will be done by the sql-cli.